allure-generator/src/main/javascript/helpers/attachment-uri.js

```diff
@@ -0,0 +1,11 @@
+import {SafeString} from 'handlebars/runtime';
+
+export default function attachmentUri(source) {
+  let sourceUri;
+  if (source.startsWith('http')) {
```

Comment:
startsWith doesn't exist in Internet Explorer. Use `source.indexOf('http') === 0`